qiskit_nature/operators/second_quantization/fermionic_op.py

```diff
@@ -645,6 +689,55 @@ def to_normal_order(self) -> FermionicOp:
             return ret
         return FermionicOp(("", 0), self.register_length, "sparse")
 
+    def normal_ordered(self) -> FermionicOp:
+        """Convert to the equivalent operator with normal order.
+
+        Returns a new operator (the original operator is not modified).
+        The returned operator is in sparse label mode.
+
+        .. note::
+
+            This method implements the transformation of an operator to the normal ordered operator.
+            The transformation is calculated by considering all commutation relations between the
+            operators. For example, for the case :math:`\\colon c_0 c_0^\\dagger\\colon`
+            where :math:`c_0` is an annihilation operator,
+            this method returns :math:`1 - c_0^\\dagger c_0` due to commutation relations.
+            See the reference: https://en.wikipedia.org/wiki/Normal_order#Multiple_fermions.
+
+        Returns:
+            The normal ordered operator.
+        """
+        temp_display_label = self.display_format
+        self.display_format = "dense"
```

Comment:
How exactly do you rely on this?

Reply:
Apparently this affects the return value of `self.to_list`, which is used  3 lines later. Again, I simply copied the code verbatim from the original implementation of `to_normal_order`.

Reply:
Ah I didn't notice that this is simple a verbatim copy. You can leave it as is for the time being then (I guess we may revisit this as part of #610).